### ISSUE TITLE
Make it possible to distinguish public and unlisted toots (after 3.1)

### ISSUE THREAD

**Issue by revi** [mastodon/mastodon#13056]
### Pitch

Add something to display whether toot is unlisted or published.

### Motivation

After #13045 unlisted toots have share buttons. I used it as a feature (not a bug) to check if I have 'published' a toot which I should've 'unlisted'. It is confusing now and makes it impossible to distinguish them.

**Comment by Cassolotl** [comment]
Sort of a duplicate of *part* of *Show privacy status icons on posts #9788*?

**Comment by revi** [comment]
Fixed with #14123.
